src/Codeception/Step.php

```diff
@@ -173,7 +174,7 @@ protected function getClassName($argument)
     {
         if ($argument instanceof \Closure) {
             return 'Closure';
-        } elseif ((isset($argument->__mocked))) {
+        } elseif ($argument instanceof MockObject) {
             return $this->formatClassName($argument->__mocked);
```

Comment:
Ok, what if there is no `__mocked` property but an object is still MockObject?

Reply:
@DavertMik Have applied changes as requested but AppVeyor build failed

`elseif ($argument instanceof MockObject && isset($argument->__mocked)` are you happy with this?